agent/consul/server_test.go

```diff
@@ -257,6 +257,26 @@ func testACLServerWithConfig(t *testing.T, cb func(*Config), initReplicationToke
 	return dir, srv, codec
 }
 
+func testGRPCIntegrationServer(t *testing.T, cb func(*Config)) (*Server, *grpc.ClientConn) {
```

Comment:
Kind of self explanatory but the purpose here is to create a test server with a listening gRPC server configured for it.

Reply:
Nice! Thanks for pulling this out 🙇🏻‍♂️